app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1> All Contestants </h1>
+<% @contestants.each do |contestant| %>
+  <h3><%= contestant.name %></h3>
+  <p>Projects: <%= contestant.projects %></p>
```

Comment:
`contestant.projects` returns a collection of project objects (an ActiveRecord relation) so this is where you would either need to:
1. Iterate over each project to print it OR
2. Since the spec in the user story shows a comma separated list, you could instead just do `contestant.projects.pluck(:name).join(", ")`

The logic above would grab that collection of projects associated to a contestant, and then pluck out the name of each project and return those names in an array. `join` would then chain the names together in a comma separated list

Because we're selecting a specific column from the database, this is the type of logic we would want to put in a model method. We could create this method in the contestant model and then call the `project_list` method on each contestant in the view
```
# contestant.rb

def project_list
   self.projects.pluck(:name).join(", ")
end
```

If we add this to the model, we'll want to test it in a model method as well